crates/symbolicator-native/src/interface.rs

```diff
@@ -606,3 +613,118 @@ impl<'de> Deserialize<'de> for AddrMode {
         AddrMode::from_str(&s).map_err(de::Error::custom)
     }
 }
+
+/// A rule for rewriting the debug file of a module.
+///
+/// This consists of a [`Regex`] and a replacement string
+/// (in `regex` crate syntax).
+///
+/// It can be created by deserializing e.g. from JSON.
+#[derive(Debug, Clone, Deserialize)]
+struct RewriteRule(
+    #[serde(deserialize_with = "deserialize_regex")] Regex,
+    Arc<str>,
+);
```

Comment:
Any specific reason you opted to make this a tuple instead of an object? If we make this an object, we have more options in the future to easier extend the rules (e.g. maybe adding a filter for which module this rule should apply).

Reply:
Only simplicity. We could easily make this an object instead.

Reply:
Do you have a suggestion for the object's field names? I'm tending towards `from` and `to`, but not totally happy with it.

Reply:
`from`, `to` works, maybe `regex`, `replacement`. But not fully happy with either.